src/main/webapp/index.jsp

```diff
@@ -1,2 +1,2 @@
-<!-- Forward to /home servlet upon entering website -->
```

Comment:
You can also check for the domain to auto redirect to the correct page.  Prod would be "rtb-step-2020.uc.r.appspot.com".